samples/CMakeLists.txt

```diff
@@ -60,7 +60,15 @@ function(add_rocwmma_sample TEST_TARGET TEST_SOURCE)
   )
 endfunction()
 
+function(add_rocwmma_rtc_sample TEST_TARGET TEST_SOURCE)
+  add_custom_command(OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/${TEST_TARGET}"
+                     COMMAND hipcc ARGS -O3 --std=c++14 -o ${TEST_TARGET} -I${ROCWMMA_SAMPLES_INCLUDE_DIR} -I${CMAKE_CURRENT_SOURCE_DIR}/../library/include/rocwmma ${TEST_SOURCE} -ldl
```

Comment:
Need to remove the rocwmma at the end of -I${CMAKE_CURRENT_SOURCE_DIR}/../library/include/~~rocwmma~~